conda_recipes/aftereffects-25.0/recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 {% set name = "aftereffects" %}
 {% set version_partial = "25" %}
-{% set version_minor = "0" %}
+{% set version_minor = "0.1" %}
 {% set version = version_partial + "." + version_minor %}
```

Comment:
should probably add version_build instead of overloading the "minor" with additional numbers. ie

```
{% set version_major = "25" %}
{% set version_minor = "0" %}
{% set version_build = "1" %}
{% set version = version_major + "." + version_minor + "." + version_build %}
```

Reply:
Good call, will revise!